### PR TITLE
SIDE-1004 - Fix ESLint warnings

### PR DESCRIPTION
**Pull request by dmorrison-olive**
https://crosschx.atlassian.net/browse/SIDE-1004

Warnings from ESLint are creating noise in LDK PRs. See [this PR](https://github.com/open-olive/loop-development-kit/pull/13/files) for an example.

### CODE REVIEW

**Review by BillyMurrayOlive — APPROVED** [review]

**Comment by BillyMurrayOlive** [comment]
Thanks for taking the time to cleanup these warnings.

**Review by iangallOlive — APPROVED** [review]

File: ldk/node/src/hostClients/stoppables.ts [hunk 1]
```diff
@@ -1,15 +1,11 @@
 /**
  * The StreamListener type is a function provided to start a stream. It accepts two parameters and returns nothing.
  *
- * @typeParam T - The type of the messages sent.
```
Comment (richseviora) [inline ldk/node/src/hostClients/stoppables.ts:4]: Please add this back in; the `typeParam` tag is consumed by TypeDoc. If it's complaining that it doesn't recognize the tag I think we can add that in the ESLint config.

File: ldk/node/src/hostClients/stoppables.ts [hunk 2]
```diff
@@ -1,15 +1,11 @@
...
  * @param error - Null, unless an error is present.
  * @param input - The event that's being generated.
  */
 export type StreamListener<T> = (error: string | null, input?: T) => void;
 
-/**
- * @typeParam T - The type of the messages sent.
```
Comment (richseviora) [inline ldk/node/src/hostClients/stoppables.ts:11]: Add back in here too

File: ldk/node/src/hostClients/stoppables.ts [hunk 3]
```diff
@@ -19,8 +15,6 @@ export interface StoppableStream<T> {
 /**
  * The StoppableMessage interface provides access to a promise that resolves when the message completes, and
  * to the ability to stop the message.
- *
- * @typeParam T - The type of the message sent.
```
Comment (richseviora) [inline ldk/node/src/hostClients/stoppables.ts:23]: Add back in here

**Review by richseviora — COMMENTED** [review]
Some changes need to be reverted plz

File: ldk/node/src/hostClients/stoppables.ts (hunk 1 above)
Comment (dmorrison-olive) [reply ldk/node/src/hostClients/stoppables.ts:4]: Gotcha - sorry I already merged this in before you commented! I see where the `@template` tag [is supported](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#template), so I'll create a separate PR with that. I'm testing it now, and that seems to pass ESLint. Sound good?

**Pull request merged**